charts/nexus-iq/values.yaml

```diff
@@ -144,17 +147,20 @@ configYaml:
     requestLog: 
       appenders:
         #All appenders set to console
-        - type: console    
```

Comment:
previously, our servers dumped all this logging to the console to be viewed as kubectl logs. Does directing the logs to files disable some monitoring that a k8s user would have expected?

Reply:
tested, and i see that i still see logs on console, so i guess that's good.